patches/tModLoader/Terraria/ModLoader/EquipLoader.cs

```diff
@@ -80,12 +87,12 @@ public static class EquipLoader
 			//Sets
 			LoaderUtils.ResetStaticMembers(typeof(ArmorIDs), true);
 			WingStatsInitializer.Load();
-			
+
```

Comment:
accidental removal of spaces?

Reply:
Hmm. This is a swap from somebody hitting tab 3+ times to just a new line.
Accidental, probably, but I think is better to use just a new line and not new line + multiple tabs.